bucket/kicad.json

```diff
@@ -1,37 +1,32 @@
 {
-    "version": "6.0.11",
+    "version": "7.0.0",
     "description": "Electronics Design Automation Suite",
     "homepage": "https://www.kicad.org",
     "license": "GPL-3.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://kicad-downloads.s3.cern.ch/windows/stable/kicad-6.0.11-x86_64.exe#/dl.7z",
-            "hash": "41d89f8bd141be2efce68f667f6cbf030191198fa517fd189f4ba6dc129e4ac7"
-        },
-        "32bit": {
-            "url": "https://kicad-downloads.s3.cern.ch/windows/stable/kicad-6.0.11-i686.exe#/dl.7z",
-            "hash": "9d128e9c09783b2c97142d6cba15cc5866a8f6ff3680b13f9578bed7dc4fbeb1"
+            "url": "https://github.com/KiCad/kicad-source-mirror/releases/download/7.0.0/kicad-7.0.0-x86_64.exe#/dl.7z",
+            "hash": "6b32bfce38bb03852cd25fe9c2174bf80091cb130d3721475fd14b2f0bfc8f8f"
         }
     },
     "pre_install": "Remove-Item \"$dir\\`$*\" -Recurse",
-    "bin": "bin\\kicad.exe",
+    "bin": [
+        "bin\\kicad.exe",
+        "bin\\kicad-cli.exe"
+    ],
```

Comment:
```suggestion
    "bin": "bin\\kicad-cli.exe",
```

Reply:
I see the rational behind removing the graphical program from the system path. But it is still possible that users might expect to be able to launch the program from the command line with `kicad`, similar to `code` or other such programs.

Reply:
Does the kicad.exe binary support commandline arguments? That's the criteria for inclusion (i.e. `code` supports arguments).

Reply:
oh. it doesn't seem to accept command line arguments